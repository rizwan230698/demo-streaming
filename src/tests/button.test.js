import React from 'react';
import { render } from 'enzyme';
import Button from '../components/button';

test('Should render a Button Component', () => {
  expect(render(<Button />)).toMatchSnapshot();
});
