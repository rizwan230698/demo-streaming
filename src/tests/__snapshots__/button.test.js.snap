// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Button Component 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "ant-btn sc-AxjAm eQZDCe",
      "type": "button",
    },
    "children": Array [],
    "name": "button",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
      "type": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
      "type": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
